My htnl input project

Dflexcee opened the pull request.

Dflexcee committed:
This my html form project

Dflexcee committed:
Update README.md

i corrected README message

Dflexcee committed:
Update README.md

STILL CHANGES

Dflexcee committed:
i added my linters

Dflexcee committed:
Merge branch 'my-htnl-input-project' of https://github.com/Dflexcee/HTML-Forms into my-htnl-input-project

Dflexcee committed:
Update .stickler.yml

i enabled my fixer

Dflexcee committed:
Update .stickler.yml

Dflexcee committed:
Update .stickler.yml

Dflexcee committed:
Update main.css

Dflexcee committed:
Update .stickler.yml

Dflexcee committed:
Update main.css

Dflexcee committed:
Update main.css

shshamim63 commented on index.html:
```diff
@@ -0,0 +1,105 @@
+                            Email address:<br>
+                            <input type="email" name="username"><br>
+                            Phone (recommended):<br>
+                            <input type="number" name="no"> <br>
```
For phone number the type of the input field is tel.

shshamim63 commented on index.html:
```diff
@@ -0,0 +1,105 @@
+                            Phone (recommended):<br>
+                            <input type="number" name="no"> <br>
+                            Standard call, messaging or data rates may apply. <br> <br>
+                            Password:<br>
```
instead of the lines you have add like `Email address: <br>` , you should use label for that.

shshamim63 commented on index.html:
```diff
@@ -0,0 +1,105 @@
+                    <br>
+                    <br>
+
+                    <div class="heading-form1" style="border: dotted 1px; background: rgb(236,238,241);;">
```
Please remove inline css from this line.

shshamim63 reviewed (commented):
### Status :arrow_forward: [Required Changes]
___
Hi @Dflexcee :wave: ,
First thing, As far i know this project is a collaborative project. But you are doing it alone. May i know, why it is like that?
You have done a great job on this project :clap: , But there are some issues that are actually holding me to declare this project as complete.
#### Please follow the instructions below to complete this project.
___
### [Mandatory changes]
---
1. The type for the phone input is tel. So please fix it.
2. Please remove inline CSS from your HTML file.
3. The font size of footer is large compare to the form. I believe you can make this more appealing.
![form1](https://user-images.githubusercontent.com/38625956/68067090-435d6400-fd6c-11e9-97a8-0de291a2226c.png)
___

#### Please fix all the mentioned issue and submit another review request.
___
### Happy Coding :cake: 
#### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawat-hossain-7750b0127/) | [Slack](Shakhawat Hossain)

Dflexcee committed:
Update main.css

i corrected the css and html issues detected by the TSE.

Dflexcee committed:
Update main.css

Dflexcee committed:
Update main.css

Dflexcee committed:
Update index.html

i corrected the  inline styling i used,
i was instructed by TSE to do remove  that as one of the requirement

shshamim63 commented:
### Status Approved :tada: 
___
Hi @Dflexcee  :wave: ,
Well done, you have complete this project successfully. :clap: 
#### This :+1:  is from me, and you can now merge the PR. Don't forget to fill the project completion form.
___
### Happy Coding
#### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawat-hossain-7750b0127/) | [Slack](Shakhawat Hossain)

Dflexcee committed:
Update index.html

i also added label to as instructed as newer way to name my input.
